kube-config-files/organisations-rw-neo4j-red-service.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: organisations-rw-neo4j-red
+  labels:
+    app: organisations-rw-neo4j-red
+    visualize: "true"
+spec:
+  replicas: 1
```

Comment:
Is there a reason why we should have a single instance here? If it is, please add a comment.